src/StoreKeeper/WooCommerce/B2C/Backoffice/Pages/Tabs/TaskLogsTab.php

```diff
@@ -259,6 +259,23 @@ public function renderTaskStatus($value, $task)
                 if (isset($errorOutput['exception-reference'])) {
                     echo __('Error reference', I18N::DOMAIN).': '.esc_html($errorOutput['exception-reference']).':<br>';
                 }
+
+                if (isset($errorOutput['exception-difference'])) {
+                    echo ''.__('Extra metadata', I18N::DOMAIN).':<br>';
+                    $extrasDifferences = $errorOutput['exception-difference'];
+                    $shopExtras = $extrasDifferences['shop-extras'];
+                    $backofficeExtras = $extrasDifferences['backoffice-extras'];
+                    if (count($shopExtras) > 0) {
```

Comment:
should we sort `$shopExtras` and `$backofficeExtras` it by wp_row_id here ? it would make the visual comparison easier

Reply:
Sorted by row id.